virttest/utils_test/libvirt.py

```diff
@@ -2525,6 +2526,34 @@ def update_memballoon_xml(vmxml, membal_dict):
     vmxml.sync()
 
 
+def create_tpm_xml(params):
+    """
+    Create xml for tpm device
+
+    :param params: tpm parameter dict
+    :return: tpm device
+    """
+    tpm_model = params.get("tpm_model", 'tpm-crb')
+    backend_type = params.get("backend_type")
+    backend_version = params.get("backend_version")
+    encryption_secret = params.get("encryption_secret")
+    device_path = params.get("device_path")
+
+    tpm_dev = tpm.Tpm()
+    tpm_dev.tpm_model = tpm_model
+    if backend_type:
+        tpm_backend = tpm_dev.Backend()
+        tpm_backend.backend_type = backend_type
+        if backend_version:
+            tpm_backend.backend_version = backend_version
+        if encryption_secret:
+            tpm_backend.encryption_secret = encryption_secret
+        if device_path:
+            tpm_backend.device_path = device_path
+        tpm_dev.backend = tpm_backend
+    return tpm_dev
```

Comment:
the return value is tpm_dev.xml or tpm_dev?

Reply:
tpm_dev

Reply:
the method name is :create_tpm_xml ,suggest syn up with return value to avoid confusion

Reply:
updated。